build/release.sh

```diff
@@ -390,19 +390,23 @@ elif [ $PLATFORM_ID -eq 6 ] || [ $PLATFORM_ID -eq 8 ]; then
 elif [ $PLATFORM_ID -eq 10 ]; then
     # Configure
     cd ../modules
-    DEBUG_BUILD="y"
     MODULAR="y"
+    declare -a debugBuildOptions=("y" "n")
 
-    # Compose, echo and execute the `make` command
-    MAKE_COMMAND="make -s clean all PLATFORM_ID=$PLATFORM_ID COMPILE_LTO=n DEBUG_BUILD=$DEBUG_BUILD MODULAR=$MODULAR USE_SWD_JTAG=$USE_SWD_JTAG USE_SWD=n APP=tinker"
-    echo $MAKE_COMMAND
-    eval $MAKE_COMMAND
+    for debugBuildOption in ${debugBuildOptions[@]}; do
```

Comment:
Is this really Electron only logic?

Reply:
Yes, because it's the only platform built by default with DEBUG_BUILD=y for the release build.